### PR TITLE
add gfapy 1.0.0rc9 recipe

### PR DESCRIPTION
**Pull request by ggonnella**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gfapy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - tests
+
+  requires:
+    - nose
+
+about:
+  home: https://github.com/ggonnella/gfapy
+  license: ISC License (ISCL)
+  summary: 'Library for handling data in the GFA1 and GFA2 formats'
+  license_family: OTHER
```
Comment (tomkinsc) [inline recipes/gfapy/meta.yaml]: Looks like the Travis build is failing at least in part due to the `license_family`. Try changing it from `OTHER` to `Other`.

**Comment by ggonnella** [comment]
Can somebody help with the error messages?

**Comment by bgruening** [comment]
Seems that this package is only available for python3. I skiped builds for python2.

**Comment by ggonnella** [comment]
@bgruening yes it is python3 only. Thank you for the fix.

**Comment by bgruening** [comment]
Your welcome!

**Comment by johanneskoester** [comment]
Hi @ggonnella! Great to see you here!!!

**Comment by ggonnella** [comment]
@johanneskoester: bioconda is very interesting, good idea!

**Pull request merged**